src/algebra/invertible.lean

```diff
@@ -73,6 +73,10 @@ by simp [mul_assoc]
 lemma inv_of_eq_right_inv [monoid α] {a b : α} [invertible a] (hac : a * b = 1) : ⅟a = b :=
 left_inv_eq_right_inv (inv_of_mul_self _) hac
 
+lemma invertible_unique {α : Type u} [monoid α] (a b : α) (h : a = b) [invertible a] [invertible b] :
```

Comment:
This is an unrelated change that is a legacy, from an earlier version where I explored using the `invertible` class instead of `units`, which I felt was worth adding anyway.

Reply:
There's a similar statement in https://github.com/leanprover-community/mathlib/blob/0736c95/src/algebra/invertible.lean#L76:
```
instance [monoid α] (a : α) : subsingleton (invertible a)
```

Reply:
Well logically it doesn't of course. I just found this a slightly more convenient form while I was messing around with defining `matrix.nonsing_inv` using `invertible` so I broke it out to a lemma. I think it would be totally reasonable to drop it though, especially as I don't even use it anyway.